Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -2703,9 +2703,12 @@ static WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
         formDataReference = IPC::FormDataReference { WTFMove(resolvedFormData) };
     }
 
-    auto eventTypes = { WebExtensionEventListenerType::WebRequestOnBeforeRequest, WebExtensionEventListenerType::WebRequestOnBeforeSendHeaders, WebExtensionEventListenerType::WebRequestOnSendHeaders };
```

Comment:
If you wanted to, you could change 'auto' to 'EventListenerTypeSet', and then nothing else would need to change. (It's safe to copy a HashSet because it's a value type, not a view type.)

Reply:
Yes, that was my original plan. However, `wakeUpBackgroundContentIfNecessaryToFireEvents` takes an r-value reference and the lambda needs to capture it too. The move and capture would be on the same line.